Assets/Scripts/Movement.cs

```diff
@@ -81,9 +98,124 @@ private void FixedUpdate()
             Velocity.y = -2f;
         }
 
-     
+        findWhipPoint();
+    }
+
+    private void ChargingTorch(InputAction.CallbackContext obj)
+    {
+        
+        print("Holding");
+        isTorchButtonHeldDown = true;
+       
+    }
+
+    private void WhileChargingTorchThrow()
+    {
+        if (isTorchButtonHeldDown)
+        {
+            Vector3 CurrentTorchPosition = HeldTorch.transform.localPosition;
+            HeldTorch.transform.localPosition = new Vector3(CurrentTorchPosition.x, CurrentTorchPosition.y - 0.1f, CurrentTorchPosition.z);
+        } else
+        {
+            HeldTorch.transform.localPosition = DefaultTorchPosition;
+        }
+    }
+
+    private void TossTorch(InputAction.CallbackContext obj)
+    {
+            print(obj.duration);
```

Comment:
I would prefer to remove debug print things like this, especially if they don't have any other descriptive text

Reply:
done